README.md

```diff
@@ -145,3 +145,23 @@ GO111MODULE="on" go get sigs.k8s.io/kind@v0.7.0
 ```
 
 Will start a kind cluster and run the end-to-end tests in `./e2e`.  To remove the cluster after running the tests, pass the `--rm` argument to the script, or run `kind delete cluster --name k8s-image-puller-e2e`.
+
+## Scratch Images
+
+Normally, the image puller cannot pull scratch images, as they do not contain a `sleep` command.
+
+But as of [2021-05-06](https://github.com/che-incubator/kubernetes-image-puller/commit/662f9817d0240043616531d3a2b180a5423c726d), the [latest](https://github.com/che-incubator/kubernetes-image-puller/commit/e28a7fb2b246d43ffb48c8f6fe35090f60c3902c) image puller builds contain a golang-based `sleep` binary that can be copied to your scratch image so that it can then be pulled.
```

Comment:
I doubt we'll be updating what "latest" refers to :) 
```suggestion
But as of [2021-05-06](https://github.com/che-incubator/kubernetes-image-puller/commit/662f9817d0240043616531d3a2b180a5423c726d), image puller builds contain a golang-based `sleep` binary that can be copied to your scratch image so that it can then be pulled.
```